src/lib/utils.js

```diff
@@ -65,13 +65,39 @@ export const loadStorage = () => {
   }
 }
 
+/**
+ * Returns a object with RGBA.
+ * @param {string} value RGBA
+ * https://stackoverflow.com/a/11003212/11321732
+ */
+const getColorObject = value => {
+  if ( [ 'none', 'undefined', null ].includes( value ) ) return null
+  const rgba = value.match( /\d+/g )
+  const rgbaObject = { r: rgba[0], g: rgba[1], b: rgba[2], a: rgba[3] }
+  return rgbaObject
```

Comment:
Destructure and save lives.

```suggestion
  if ( [ 'none', 'undefined', null ].includes( value ) ) return null

  const [ r, g, b, a ] = value.match( /\d+/g )
  return { r, g, b, a }
```